.circleci/config.yml

```diff
@@ -199,37 +199,97 @@ jobs:
   python_integration:
     docker:
       - image: "circleci/python:3.6.1"
+        environment:
+          ETL_SECRET_KEY: test_key
+          EXTRACT_ENDPOINT_STORAGE_DOMAIN: http://127.0.0.1:10000/devstoreaccount1
+          EXTRACT_ENDPOINT_STORAGE_ACCOUNT: devstoreaccount1
+          EXTRACT_ENDPOINT_STORAGE_KEY: Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==
+          EXTRACT_ENDPOINT_CONTAINER: energuide-extracted-data
       - image: "mongo:3.6.2"
+      - environment: executable=blob
+        image: arafato/azurite
     steps:
       - checkout:
           path: ~/repo
       - restore_cache:
-          key: "deps1-{{ .Branch }}-{{ checksum \"requirements.txt\" }}"
+          key: "deps1-{{ .Branch }}-{{ checksum \"etl/requirements.txt\" }}"
+      - restore_cache:
+          key: "deps2-{{ .Branch }}-{{ checksum \"extract_endpoint/requirements.txt\" }}"
       - run:
           command: |
               sudo apt-get install apt-transport-https
               sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2930ADAE8CAF5059EE73BB4B58712A2291FA4AD5
               echo "deb http://repo.mongodb.org/apt/debian jessie/mongodb-org/3.6 main" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.6.list
               sudo apt-get update
-              sudo apt-get install -y mongodb-org-tools=3.6.2
+              sudo apt-get install -y mongodb-org-tools=3.6.2 mongodb-org-shell=3.6.2
+              echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ wheezy main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
+              sudo apt-key adv --keyserver packages.microsoft.com --recv-keys 52E16F86FEE04B979B07E28DB02C46DF417A0893
+              sudo apt-get install apt-transport-https
+              sudo apt-get update && sudo apt-get install azure-cli
           name: "Installing packages"
       - run:
           command: |
+              echo $EXTRACT_ENDPOINT_STORAGE_DOMAIN
+              echo $EXTRACT_ENDPOINT_STORAGE_ACCOUNT
+              echo $EXTRACT_ENDPOINT_STORAGE_KEY
+              echo $EXTRACT_ENDPOINT_CONTAINER
```

Comment:
why the echos?

Reply:
because I totally forgot about them 😂 thanks! 🔥